sanic/config.py

```diff
@@ -1,10 +1,75 @@
+from sanic.defaultFilter import DefaultFilter
 import os
-
 import types
 
 SANIC_PREFIX = 'SANIC_'
 
 
+DEFAULT_LOG_CONF = {
```

Comment:
But when use with `app.config.from_object` method and etc it should be called `LOGGING` instead of `DEFAULT_LOGGING`.